src/main/java/seedu/address/model/booking/Name.java

```diff
@@ -3,25 +3,25 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
+import seedu.address.model.name.CommonName;
+
 /**
  * Represents a Person's name in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidName(String)}
  */
-public class Name {
```

Comment:
Can we refactor this class to be named as VisitorName? similar to how we did for Residence by calling it ResidenceName?